### PR TITLE
NewRelic ErrorBreakdown & WebsiteFunnel widgets

### PR DESCRIPTION
**Pull request by milenvoutchev**
NewRelic ErrorBreakdown & WebsiteFunnel widgets; Tests;

### CODE REVIEW

File: server/src/routes/newRelicRoute.js [hunk 1]
```diff
@@ -99,4 +99,42 @@ router.get('/cliErrors', (req, res, next) => {
...
+  if (cachedPayload) {
+    res.json(getResponseSuccess(cachedPayload));
+    return;
+  }
+
+  newRelicService.getWebsiteFunnel()
+      .then((payload) => {
+        cacheService.set(cacheKey, payload);
+        res.json(getResponseSuccess(payload));
+      })
```
Comment (artemstepanenko) [inline server/src/routes/newRelicRoute.js]: We can implement a helper method for these two lines, so we'll have `.then(payload => handleSuccessfulResponse(res, cacheKey, payload))`.

File: server/src/routes/newRelicRoute.js [hunk 2]
```diff
@@ -99,4 +99,42 @@ router.get('/cliErrors', (req, res, next) => {
...
+      .catch((err) => {
+        next(err);
+      });
```
Comment (artemstepanenko) [inline server/src/routes/newRelicRoute.js]: We can make it a one-liner: `.catch(err => next(err))`

File: server/src/service/newRelicService.js [hunk 3]
```diff
@@ -82,6 +82,38 @@ class NewRelicService {
         current: _get(insightsResponse, 'results[0].count'),
       }));
   }
+
+  getErrorBreakdown() {
+    const nrql = 'SELECT count(*) from TransactionError WHERE appName = \'www.campsy.de\' SINCE 1 day ago FACET `error.class`';
+
+    return this.getQueryResponse(nrql)
+      .then((insightsResponse) => {
```
Comment (artemstepanenko) [inline server/src/service/newRelicService.js]: If there's only one param, no need to use parenthesis.

**Comment by milenvoutchev** [comment]
@artemstepanenko Fixed.
Regarding `When the browser window width is reduced, and Error Breakdown charts become tables, they don't contain any data.` this will be fixed in https://github.com/rocket-internet-berlin/RocketDashboard/tree/mv/extract-base-components

**Pull request merged**